### ISSUE TITLE
android not working tab

### ISSUE THREAD

**Issue by isthaison** [kmagiera/react-native-gesture-handler#241]
<TapGestureHandler
                    {...Platform.select({
                      ios: {
                        maxDeltaY: 0,
                        maxDeltaX: 0,
                      },
                      android: {
                        maxDeltaY: 0,
                        maxDeltaX: 0,
                      },
                    })}
                    onHandlerStateChange={e => this._onSingleTap(e, 0)}>

**Comment by osdnk** [comment]
Thank you for the issue
Hope will be fixed soon

**Comment by isthaison** [comment]
Thank's you. let me try again!

**Comment by osdnk** [comment]
@isthaison, I can't understand you.
I suppose this issue has been fixed so I decided to close it.
If you still face the problem, reopen the issue or write here.
